dev/planning-example.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# ============================
+# Observation Planning Toolbox
+# ============================
+"""
+The observation planning toolbox consists of a number of entities that
+simplify the planning and scheduling of observations by abstracting much
+of the boilerplate code necessary to accomplish the same thing using
+core astropy or ephemerides objects.  These can be thought of as common
+idioms or convenience classes and functions that would be recreated over
+and over by astronomers and or software engineers building telescope
+scheduling software.
+
+Important assumptions:
+
+- It should be convenient to specify dates and times in the local timezone
+  of the observer.
+
+- It should be convenient to do timezone conversions easily.
+
+  This means methods that can take dates can either take astropy
+  dates (as used internally) or python datetime objects (not naive, but
+  tagged with timezone).  So dates can be passed in as timezone-aware
+  datetime objects and results can be easily converted between such.
+
+"""
+
+# ================
+# Observer objects
+# ================
+"""
+An observer defines a single terrestrial location and applicable
+environmental attributes thereof that affect observation calculations
+at that location.
+"""
+# Define an observer by full specification.
+# Latitude/Longitude take input as strings or quantities,
+# elevation, pressure, temperature must be quantities.
+# Accept `timezone` from pytz
+from astroplan import Observer
+import astropy.units as u
+import pytz
+
+obs = Observer(name='Subaru Telescope',
+               longitude='-155:28:48.900',
```

Comment:
@adrn I think in my last conversations with somebody (?) I was convinced that accepting strings is "how you'd expect it to behave" and in most cases it's shorter to write. But the more I think about it, the more I want to do it your way, so that you could specify a quick example in two short lines: 

```python
location = EarthLocation(0, 0, 0)
obs = Observer(name='Water', location=location)
```